photon-core/src/main/java/org/photonvision/vision/frame/provider/USBFrameProvider.java

```diff
@@ -31,29 +36,71 @@ public class USBFrameProvider extends CpuImageProcessor {
     @SuppressWarnings("SpellCheckingInspection")
     private VisionSourceSettables settables;
 
+    private long lastTime = 0;
+
     @SuppressWarnings("SpellCheckingInspection")
     public USBFrameProvider(CvSink sink, VisionSourceSettables visionSettables) {
         logger = new Logger(USBFrameProvider.class, sink.getName(), LogGroup.Camera);
 
         cvSink = sink;
         cvSink.setEnabled(true);
         this.settables = visionSettables;
+
+        SmartDashboard.putBoolean("/photonvision/use_old_cscore_behavior", false);
     }
 
     @Override
     public CapturedFrame getInputMat() {
-        // We allocate memory so we don't fill a Mat in use by another thread (memory model is easier)
-        var mat = new CVMat();
-        // This is from wpi::Now, or WPIUtilJNI.now(). The epoch from grabFrame is uS since
-        // Hal::initialize was called
-        long captureTimeNs = cvSink.grabFrame(mat.getMat()) * 1000;
-
-        if (captureTimeNs == 0) {
-            var error = cvSink.getError();
-            logger.error("Error grabbing image: " + error);
-        }
+        if (SmartDashboard.getBoolean("/photonvision/use_old_cscore_behavior", false)) {
```

Comment:
We need to expose this switch, at least temporarily, somewhere. Is a `retained` nt boolean OK? Do I need to add this to PhotonConfiguration? WDYT